nodejs/week1/warmup.js

```diff
@@ -0,0 +1,32 @@
+class Circle {
+    constructor(radius) {
+        this.radius = radius;
+    }
+
+    getDiameter() {
```

Comment:
By the implementation, it's rather not `get`Diameter, but `print`Diameter 🙂. Consider returning values instead of printing them.

Reply:
Also, there is no reason to store `diameter`, `circumference` and `area`, as they are easily reproducible based on `radius`.